modules/client-configuration/pages/supported-features.adoc

```diff
@@ -111,12 +111,22 @@ The operating system you run on a client is supported by the organization that s
 | {check}
 
 | {centos} 8
+ifeval::[{uyuni-content} == true]
+| {x86_64}, {ppc}
+endif::[]
+ifeval::[{suma-content} == true]
 | {x86_64}
+endif::[]
 | {cross}
 | {check}
 
 | {centos} 7
+ifeval::[{uyuni-content} == true]
+| {x86_64}, {ppc}
+endif::[]
+ifeval::[{suma-content} == true]
 | {x86_64}
+endif::[]
```

Comment:
Did you check that this builds OK?

Reply:
See the attached PDFs. They look OK.

Reply:
Since the PDFs were able to build, I will merge.